package.json

```diff
@@ -26,8 +26,8 @@
   "scripts": {
     "build": "gulp build",
     "changelog": "auto-changelog -p",
-    "lint": "eslint --ext .js . && stylelint \"**/*.{css,less,overrides,variables}\"",
-    "lint-fix": "eslint --ext .js . --fix && stylelint \"**/*.{css,less,overrides,variables}\" --fix"
+    "lint": "eslint --ext .js . && stylelint \"**/*.{css,less,overrides,variables}\" && prettier --loglevel warn \"!dist\" \"!test/coverage\" \"**/*.XX{css,less,overrides,variables}\" --check",
```

Comment:
TODO no `XX` before merge, currently, Prettier is disabled until all CS problems are fixed